airflow/deploy.sh

```diff
@@ -4,7 +4,12 @@ BUCKET="dse-infra-dev-us-west-2-mwaa"
 ENVIRONMENT="dse-infra-dev-us-west-2-mwaa-environment"
 
 # Copy the dags
-aws s3 sync dags s3://${BUCKET}/dags --delete --exclude "*__pycache__*"
+for SUBDIR in $(ls dags); do
```

Comment:
This is a little less aggressive about `--delete`ing things in the `dags` directory: now we only delete things that belong to this repo's subdirectories.

This change is because I'm including some DAGs from other repos in the Airflow instance. It may not be considered best practice, or what we would have designed from scratch, but was expedient.

Perhaps worth having a discussion around multi-repo orchestrators?